### PR TITLE
Add support for /paths/strict-send

### PR DESCRIPTION
**Pull request by tamirms**
Fixes https://github.com/stellar/java-stellar-sdk/issues/247

In the v0.20.0 release, Horizon added a new endpoint for finding path payments between a source asset and a set of destination assets. This commit adds support for querying this new path finding endpoint.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 As this project is pre 1.0, breaking changes may happen for minor version bumps. A breaking change will get clearly notified in this log.
 
+## 0.11.0
+
+* Rename `org.stellar.sdk.requests.PathsRequestBuilder` to `org.stellar.sdk.requests.StrictReceivePathsRequestBuilder`
```
Comment (abuiles) [inline CHANGELOG.md]: @tamirms shall we keep `PathsRequestBuilder` as an alias to `StrictReceivePathsRequestBuilder`?

File: src/main/java/org/stellar/sdk/requests/StrictSendPathsRequestBuilder.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    super(httpClient, serverURI, "");
+    this.setSegments("paths", "strict-send");
+  }
+
+  public StrictSendPathsRequestBuilder destinationAccount(String account) {
+    uriBuilder.setQueryParameter("destination_account", account);
+    return this;
+  }
+
+  public StrictSendPathsRequestBuilder destinationAssets(List<Asset> assets) {
```
Comment (abuiles) [inline src/main/java/org/stellar/sdk/requests/StrictSendPathsRequestBuilder.java]: can we add a check to make sure `destionation_account` hasn't been set already?

File: src/main/java/org/stellar/sdk/requests/StrictSendPathsRequestBuilder.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+import java.util.List;
+
+public class StrictSendPathsRequestBuilder extends RequestBuilder {
+  public StrictSendPathsRequestBuilder(OkHttpClient httpClient, HttpUrl serverURI) {
+    super(httpClient, serverURI, "");
+    this.setSegments("paths", "strict-send");
+  }
+
+  public StrictSendPathsRequestBuilder destinationAccount(String account) {
+    uriBuilder.setQueryParameter("destination_account", account);
```
Comment (abuiles) [inline src/main/java/org/stellar/sdk/requests/StrictSendPathsRequestBuilder.java]: same here, can you check that `destination_assets` has not been set.

File: CHANGELOG.md (hunk 1 above)
Comment (tamirms) [reply CHANGELOG.md]: I think renaming the class will make it consistent with https://github.com/stellar/java-stellar-sdk/pull/252

**Pull request merged**